cpp/ql/src/semmle/code/cpp/Declaration.qll

```diff
@@ -139,6 +139,19 @@ class Declaration extends Locatable, @declaration {
     this.hasQualifiedName("std", "", name)
   }
 
+  /**
+   * Holds if this declaration has the given name in the global namespace,
+   * the `std` namespace or the `bsl` namespace.
+   * We treat `std` and `bsl` as the same in a bunch of our models.
+   */
+  predicate hasGlobalOrStdishName(string name) {
```

Comment:
I'm not all that keen on the term "Stdish" ... though I appreciate we don't want "GlobalStdOrBsl" as that precludes some day adding a third namespace to the predicate.  Thoughts?

Reply:
I can understand that. I just didn't think that `hasGlobalOrStdOrBslName` wouldn't be any better.
On the other hand, we already do keep the distinction between `hasGlobalOrStdName` and this new predicate, so we should probably be clearer with the name and include bsl there.
Otherwise, people reading the code have to maintain the additional overhead of looking up what `Stdish` exactly means.
I'll make the change, here and in the other PRs.

Reply:
Seems reasonable. :+1: